aws/table_aws_ecs_cluster.go

```diff
@@ -0,0 +1,283 @@
+				Description: resourceInterfaceDescription("tags"),
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getAwsEcsClusterTags,
+				Transform:   transform.From(getAwsEcsClusterTurbotTags),
+			},
+		}),
+	}
+}
+
+//// LIST FUNCTION
+
+func listEcsClusters(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	var region string
+	matrixRegion := plugin.GetMatrixItem(ctx)[matrixKeyRegion]
+	if matrixRegion != nil {
+		region = matrixRegion.(string)
+	}
+
+	// Create Session
+	svc, err := EcsService(ctx, d, region)
+	if err != nil {
+		return nil, err
+	}
+	params := &ecs.ListClustersInput{}
+	pagesLeft := true
+
+	for pagesLeft {
+		result, err := svc.ListClusters(params)
+		if err != nil {
+			return nil, err
+		}
+
+		for _, results := range result.ClusterArns {
+			plugin.Logger(ctx).Trace("clusterArnclusterArnclusterArn", "clusterArn", results)
+			d.StreamListItem(ctx, &ecs.Cluster{
+				ClusterArn: results,
+			})
+		}
+
+		if result.NextToken != nil {
+			pagesLeft = true
+			params.NextToken = result.NextToken
+		} else {
+			pagesLeft = false
+		}
+	}
+
+	return nil, err
+}
+
+/// HYDRATE FUNCTIONS
+
+func getEcsCluster(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	logger := plugin.Logger(ctx)
+	logger.Trace("getEcsCluster")
+	var region string
+	matrixRegion := plugin.GetMatrixItem(ctx)[matrixKeyRegion]
+	if matrixRegion != nil {
+		region = matrixRegion.(string)
+	}
+
+	var clusterArn string
+	if h.Item != nil {
+		clusterArn = *h.Item.(*ecs.Cluster).ClusterArn
+	} else {
+		quals := d.KeyColumnQuals
+		clusterArn = quals["cluster_arn"].GetStringValue()
+	}
+
+	// Create Session
+	svc, err := EcsService(ctx, d, region)
+	if err != nil {
+		return nil, err
+	}
+
+	params := &ecs.DescribeClustersInput{
+		Clusters: []*string{aws.String(clusterArn)},
+	}
+
+	op, err := svc.DescribeClusters(params)
+	if err != nil {
+		logger.Debug("getEcsCluster", "ERROR", err)
+		return nil, err
+	}
+
+	if op.Clusters != nil && len(op.Clusters) > 0 {
+		return op.Clusters[0], nil
+	}
+
+	return nil, nil
+}
+
+func getAwsEcsClusterTags(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	plugin.Logger(ctx).Trace("getAwsEcsClusterTags")
+	var region string
+	matrixRegion := plugin.GetMatrixItem(ctx)[matrixKeyRegion]
+	if matrixRegion != nil {
+		region = matrixRegion.(string)
+	}
+	clusterArn := *h.Item.(*ecs.Cluster).ClusterArn
+
+	// Create service
+	svc, err := EcsService(ctx, d, region)
+	if err != nil {
+		return nil, err
+	}
+
+	params := &ecs.ListTagsForResourceInput{
+		ResourceArn: &clusterArn,
+	}
+
+	clusterdata, err := svc.ListTagsForResource(params)
+	if err != nil {
+		return nil, err
+	}
+
+	return clusterdata, nil
+}
+
+//// TRANSFORM FUNCTIONS 
+
+func getAwsEcsClusterTurbotTags(ctx context.Context, d *transform.TransformData) (interface{},
+error) {
+	ecsClusterTags := d.HydrateItem.(*ecs.ListTagsForResourceOutput)
+
```

Comment:
Return `nil`, if the `ecsClusterTags.Tags` return nil
```suggestion

if ecsClusterTags.Tags == nil {
  return nil, nil
}

```